### DISCUSSION TITLE
Expose EditorPropertyResource editors to plugins

### DISCUSSION THREAD

**Discussion by Qubus0** [godotengine/godot-proposals#7788] [Editor]
The Resource (and array/dictionary) editors are very good.
With every EditorInspectorPlugin you end up building the same editors from scratch.

It would be extremely handy if there was a way to use those editors.
For custom editors that would only require minor modifications, it would also be nice to directly get the node, so that they can be changed via code.

**Comment by AnidemDex** [comment]
Isn't https://github.com/godotengine/godot-proposals/issues/300 related?
